Automattic/jetpack
# Search: Add siteTitle to fetched data.

#### Changes proposed in this Pull Request:


Adds the siteTitle to the plugin data so we can use this in the First Run state.

![NowWithSiteTitle](https://user-images.githubusercontent.com/40267301/195599833-a026d41d-25e4-4d24-b09f-00d8ed43b6ff.png)

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Visit updated Search dashboard.
2. Open Dev Tools and toggle the `hasIndex` value to `false`.
3. Confirm the index UI now shows the title from your test site.
